vividus-plugin-web-app/src/main/resources/properties/profile/web/profile.properties

```diff
@@ -42,6 +42,7 @@ web-application.tablet-resolution-width-threshold=1024
 web-application.phone-resolution-width-threshold=640
 web-application.environment-type=test
 web-application.authentication-mode=URL
+web-application.life-scope=story
```

Comment:
Looks confusing to me. It is a driver's scope, not the application. Could it be `driver-life-scope`

Reply:
`driver` will be not clear for end-users, also I thought about `browser-life-scope`, but in this case we'll loose naming consistency

Reply:
In the current naming, it looks like the user controls an application. When he controls session

Reply:
`web-application.session-scope`
`mobile-application.session-scope`
?

Reply:
Looks good to me. Maybe it will be better to roll out voting :)